apps/docs/dictionary.txt

```diff
@@ -122,4 +122,8 @@ itemsDistribution
 fixedHeight
 groupId
 hideToc
-Dismissible
\ No newline at end of file
+Dismissible
+gi-pagination-next-btn
```

Comment:
Do you guys know why do I have to add classes in dicitonary.txt? 🤔 

@pregno @ramonmulia @BogdanDanilescu

Reply:
Because we are adding the `html` in the mdx file.

Reply:
It never complained on other HTML files using classes?

Reply:
It's checking only mdx files